wolfcrypt/src/sha256.c

```diff
@@ -54,16 +54,16 @@
     }
     int wc_Sha256Update(Sha256* sha, const byte* data, word32 len)
     {
-		if (sha == NULL ||  (data == NULL && len > 0)) {
-        	return BAD_FUNC_ARG;
+	    if (sha == NULL ||  (data == NULL && len > 0)) {
```

Comment:
There's a hard tab here.  I can tell because the indentation doesn't match what you'd expect.  If you use 'h' in vim to move left from the first character typed you'll move back spaces until you hit the hard tab, another way to "see" it.